src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -829,6 +829,24 @@
                 }
             }
         },
+        "stm_state": {
+            "id" : "stm_state",
```

Comment:
would it be possible to add in_sync offset here as well ?

Reply:
I was actually thinking about it and ended up not doing it in the first pass. The reason is it makes the snapshottable_stm interface ugly. _insync_offset is a property of persisted_stm impl where as the list of stm pointers that a partition maintains are of base class snapshottable_stm. We either have to leak that detail into snapshottable_stm or deal with individual stm types separately, both of which are not great IMO. thoughts?